src/esm_runscripts/filedicts.py

```diff
@@ -173,12 +204,13 @@ def cp(self, source: str, target: str) -> None:
         source_path = self[f"absolute_path_in_{source}"]
         target_path = self[f"absolute_path_in_{target}"]
 
-        # Checks
+        # general checks
+        # TODO (deniz): need to add higher level exception handler (eg. user_error)
```

Comment:
```suggestion
```
No please, we agreed we will only use user_error for "extended-yaml syntax" problems

Reply:
I think I had to write a better comment there. I meant that we could wrap this in a `try-except` block and exception would call the `user_error`